controllers/front/CartController.php

```diff
@@ -635,16 +675,67 @@ protected function areProductsAvailable()
         }
 
         if ($product['active']) {
+            // get the configuration about showing quantities
+            $display_quantities = Configuration::get('PS_DISPLAY_QTIES');
+
+            // Check if the display quantities option is enabled, and check quantities to provide singular or plural form as well
+            if ($display_quantities) {
```

Comment:
@marsaldev why this check? `PS_DISPLAY_QTIES` is just about showing quantities on the product page. Right? It shouldn't matter in the cart. Quantities there should always be checked unless `allow_ordering_oos` is enabled for the product.